docs/example_go.md

```diff
@@ -26,6 +26,8 @@ To skip straight to the fully optimized setup, go to this subdirectory:
 
 [Recommended Tiltfile](https://github.com/windmilleng/tilt-example-go/blob/master/3-recommended/Tiltfile)
 
+_Quick note: Tilt's first deployment of a service takes a few seconds longer than subsequent ones, due to some behind-the-scenes setup. Measurements in this guide focus on non-initial builds._
```

Comment:
Hmmmm...this note feels like it's disconnected from any of the content around this. Have you heard any comments yet about this?

How would you feel about making this a footnote attached to one of the markdown tables? I think Kramdown supports footnotes

Reply:
good call, done